EXTRA.md

```diff
@@ -0,0 +1,30 @@
+Extra Information and Tips
+======
+
+## SSL Verification
+To disable SSL verification (for example, if a server links to another)
+Add to the server_settings table in bot.db:
+
+| server_id | setting | value |
+| --- | --- | --- |
+| 1 | ssl-verify  | false |
+
+---
+
+## SASL Authentication
+Add to the server_settings table in bot.db:
+
+| server_id | setting | value |
+| --- | --- | --- |
+| 1 | sasl | {"mechanism": "PLAIN", "args": "username:password"} |
+
+---
+
+## User Identification (IRCv3)
```

Comment:
Need to mention that this only works on networks that support `WHOX` and IRCv3's `extended-join` & `account-notify`